_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,130 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] for which one of the following is true:
+
+- the element is an `img` element where at least one of the [image sources][] in its [source set][] does not reference an SVG document; or
+- the element is an `input` element in the [Image Button][] state and its `src` [attribute value][] does not reference an SVG document.
+
```

Comment:
Do we also want to add images inserted through `background-image` CSS? Maybe it's a bit hard to shoehorn that without too much twist on the language (not sure "image source" apply to these) and it would be better suited in a separate rule…

Reply:
I believe image source also applies here. An image source is a url, and the CSS image type can also be a url reference. I'll give it a go. Let me know if you think it holds up...

Reply:
Yes, that should be good 👍 
CSS is also calling it "image source" (it's the Section title in https://drafts.csswg.org/css-backgrounds-3/#background-image ) so it should be good.